client/my-sites/themes/theme-options.js

```diff
@@ -86,7 +86,14 @@ function getAllThemeOptions( { translate, isFSEActive } ) {
 			context: 'verb',
 			comment: 'label for selecting a site for which to upgrade a plan',
 		} ),
-		getUrl: ( state, themeId, siteId ) => `/checkout/${ getSiteSlug( state, siteId ) }/business`,
+		getUrl: ( state, themeId, siteId ) => {
+			const slug = getSiteSlug( state, siteId );
+			const redirectTo = encodeURIComponent(
+				`/setup/designSetup?siteSlug=${ slug }&theme=${ themeId }`
```

Comment:
Since this is a relative URL, the redirect from the pending page to `/setup/designSetup` will always use a Singe Page App redirect. If you want this to get a full page reload, just change this URL to be absolute instead. Of course, constructing an absolute URL is not trivial since the user could be on wordpress.com, wpcalypso.wordpress.com, calypso.localhost, etc. You can use a technique [like this](https://github.com/Automattic/wp-calypso/blob/f8cb8cd8f0df2f8131e3c2b5eb20e31f0cfc9b7a/client/my-sites/checkout/composite-checkout/lib/pending-page.ts#L164) to figure it out.